src/OneExplorer/OneExplorer.ts

```diff
@@ -405,45 +406,42 @@ input_path=${modelName}.${extName}
       const fstat = fs.statSync(fpath);
 
       if (fstat.isDirectory()) {
-        const childNode = new Node(NodeType.directory, [], vscode.Uri.file(fpath));
+        const dirNode = this.createDirectoryNode(fpath);
 
-        this.searchNode(childNode);
-        if (childNode.childNodes.length > 0) {
-          node.childNodes.push(childNode);
+        if (dirNode) {
+          node.childNodes.push(dirNode);
         }
       } else if (
           fstat.isFile() &&
           (fname.endsWith('.pb') || fname.endsWith('.tflite') || fname.endsWith('.onnx'))) {
-        const childNode = new Node(NodeType.baseModel, [], vscode.Uri.file(fpath));
-
-        this.searchPairConfig(childNode);
+        const baseModelNode = this.createBaseModelNode(fpath);
```

Comment:
(2) Introduce createDirectory/BaseModel/ConfigNode() helpers for readibility

By interoducing some creator functions, enhance readibility of the main code.